pytorch_lightning/trainer/trainer.py

```diff
@@ -2612,6 +2613,36 @@ def weights_summary(self, val: Optional[str]) -> None:
         rank_zero_deprecation("Setting `Trainer.weights_summary` is deprecated in v1.5 and will be removed in v1.7.")
         self._weights_summary = val
 
+    """
+    Estimator Properties
+    """
```

Comment:
Is there a plan to add others? If not let's just move it to an existing section?

Reply:
not right now. will update!